### PR TITLE
fixed multiple naming mistakes

### PR DESCRIPTION
**Pull request by MHove89**
I found multiple mistakes and inconsistent naming. 
I have no idea how this has happened. Unfortunately i only have myself to blame.

This is critical, since a few of them is actually breaking a current release. 

I quickly did a late night session, renaming and fixing, but I need the changes verified. 

 @jesperweber  Can you go through it and verify? 
I have not tested this yet!

### CODE REVIEW

File: src/Enterspeed.Source.UmbracoCms.V8/_content/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-failedjobs.controller.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     function getConfiguration() {
         vm.loadingConfiguration = true;
-        dashboardResources.getEnterspeedConfiguration()
```
Comment (MHove89) [inline src/Enterspeed.Source.UmbracoCms.V8/_content/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-failedjobs.controller.js:19]: This is broken on the current released version of the connector

File: src/Enterspeed.Source.UmbracoCms.V8/_content/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-seed.controller.js [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
     function getConfiguration() {
         vm.loadingConfiguration = true;
-        dashboardResources.getEnterspeedConfiguration()
```
Comment (MHove89) [inline src/Enterspeed.Source.UmbracoCms.V8/_content/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-seed.controller.js:14]: This breaks the current  released version of the connector

File: src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-failedjobs.controller.js [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
     function getConfiguration() {
         vm.loadingConfiguration = true;
-        dashboardResources.getEnterspeedConfiguration()
```
Comment (MHove89) [inline src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-failedjobs.controller.js:19]: This breaks the current  released version of the connector

File: src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-seed.controller.js [hunk 4]
```diff
@@ -35,7 +35,7 @@
 
     function getConfiguration() {
         vm.loadingConfiguration = true;
-        dashboardResources.getEnterspeedConfiguration()
```
Comment (MHove89) [inline src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-seed.controller.js:38]: This breaks the current  released version of the connector

**Review by jesperweber — APPROVED** [review]
Looking good and it's working :-)

**Pull request merged**